api/src/test/java/org/apache/iceberg/AssertHelpers.java

```diff
@@ -155,4 +155,18 @@ public static void assertEmptyAvroField(GenericRecord record, String field) {
         AvroRuntimeException.class,
         () -> record.get(field));
   }
+
+  /**
+   * Same as {@link AssertHelpers#assertThrowsCause}, but this method compares root cause.
+   */
+  public static void assertThrowsRootCause(String message,
+                                       Class<? extends Exception> expected,
+                                       String containedInMessage,
+                                       Runnable runnable) {
```

Comment:
Nit:   It's not the correct iceberg indent, I think you will need to configure your IDE by following this: https://iceberg.apache.org/community/#setting-up-ide-and-code-style

Reply:
fixed it.